larq/metrics.py

```diff
@@ -62,7 +63,54 @@ def get_training_metrics():
     return _GLOBAL_TRAINING_METRICS
 
 
-class FlipRatio(Metric):
+class _CompatibilityMetric(Metric):
```

Comment:
Thinking about this again what do you think about `LarqMetric`?

Reply:
Sure, at this point there is nothing that really makes it larq-specific, but since all Larq metrics will subclass it, why not